share/doc/homebrew/Interesting-Taps-&-Branches.md

```diff
@@ -36,7 +36,7 @@ Homebrew has the capability to add (and remove) multiple taps to your local inst
 
 *   [homebrew/x11](https://github.com/Homebrew/homebrew-x11): Formulae with hard X11 dependencies.
 
-`brew search` looks in these main taps and as well in [Homebrew/homebrew](https://github.com/Homebrew/homebrew). So don't worry about missing stuff. We will add other taps to the search as they become well maintained and popular.
+`brew search` looks in these main taps as well as in [Homebrew/core](https://github.com/Homebrew/homebrew-core). So don't worry about missing stuff. We will add other taps to the search as they become well maintained and popular.
```

Comment:
Hmm. Do you think we should format all the repo names on this page in fixed-width, like `homebrew/apache`, `homebrew/binary`, etc? Right now they're all highlighted because they're links, but that's incidental. I'm thinking yes.